Lib/ast.py

```diff
@@ -59,7 +59,8 @@ def literal_eval(node_or_string):
     sets, booleans, and None.
     """
     if isinstance(node_or_string, str):
-        node_or_string = parse(node_or_string, mode='eval')
+        # bpo-41887: Allow leading whitespace
```

Comment:
I don't usually like mentioning issue numbers in comments, unless there is a lot of explanation in the issue that wouldn't fit in a comment.
```suggestion
        # Strip leading whitespace
```

Reply:
If we are not going to mention about the issue, I guess we can totally remove the comment since the call `node_or_string.strip(" \t")` is explicit enough about what it is doing.